src/components/Table/Table.module.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../sharedStyles.scss";
+@import "../../colours.scss";
+
+.tableHeader {
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.controls {
+  display: flex;
```

Comment:
user this from bootstrap please and make sure that this approach is not being used anywhere please